packages/functional-tests/tests/syncV3/signIn.spec.ts

```diff
@@ -20,129 +21,176 @@ test.describe('severity-2 #smoke', () => {
       test.slow();
     });
 
-    test('verified, does not need to confirm', async ({
+    test('verified email, does not need to confirm', async ({
       target,
       credentials,
       syncBrowserPages,
     }) => {
       const email = credentials.email;
-      const { page, login, connectAnotherDevice, signinTokenCode } =
-        syncBrowserPages;
+      const { page, login, connectAnotherDevice } = syncBrowserPages;
+
       await page.goto(
         `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email`
       );
-      await login.setEmail(email);
-      await signinTokenCode.clickSubmitButton();
-      await login.setPassword(PASSWORD);
-      await login.submit();
+      await login.login(email, PASSWORD);
+
       await expect(connectAnotherDevice.fxaConnected).toBeEnabled();
     });
 
-    test('verified, resend', async ({ emails, target, syncBrowserPages }) => {
-      const { configPage, page, login, connectAnotherDevice, signinTokenCode } =
+    test('verified email, can ask to resend code', async ({
+      emails,
+      target,
+      syncBrowserPages,
+    }) => {
+      const { page, login, connectAnotherDevice, signinTokenCode } =
         syncBrowserPages;
+
       const [email] = emails;
-      const config = await configPage.getConfig();
-      test.fixme(
-        config.showReactApp.signUpRoutes,
-        'this test goes through the signup flow instead of sign in, skipping for react, see FXA-8973'
-      );
+
+      // Simulate a new sign up that requires a signin verification code.
+      target.auth.signUp(email, PASSWORD, {
+        service: 'sync',
+        preVerified: 'true',
+      });
 
       await page.goto(
         `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email`
       );
-      await login.setEmail(email);
-      await signinTokenCode.clickSubmitButton();
-      await login.setPassword(PASSWORD);
-      await login.confirmPassword(PASSWORD);
-      await login.setAge('21');
-      await login.submit();
+      await login.login(email, PASSWORD);
 
       // Click resend link
       await signinTokenCode.resendLink.click();
-      await signinTokenCode.successMessage.waitFor({ state: 'visible' });
       await expect(signinTokenCode.successMessage).toBeVisible();
       await expect(signinTokenCode.successMessage).toContainText(
         /Email re-?sent/
       );
       const code = await target.email.waitForEmail(
         email,
-        EmailType.verifyShortCode,
-        EmailHeader.shortCode
```

Comment:
Looks like this test was landing on the `confirm_signup_code` page instead of `signin_token_code` page, false positive... Using the auth method instead + checking the page path in POM as you're doing are good safeguards to make sure we're testing the intended flow 👍